### PR TITLE
chore: upgrade @grafana dependencies

### PR DESCRIPTION
**Pull request by abdullah-cognite**
Upgraded `@grafana/data`, `@grafana/ui`, `@grafana/runtime` and `@grafana/schema`  dependencies to version 11.3.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 This article documents the ongoing improvements we're making to the **Cognite
 Data Source for Grafana**.
 
+## 4.3.0 - Oct 24th, 2024
+
+### Bug fixes
+
+- Upgrade plugin dependencies to fix CVEs
+
```
Comment (polomani) [inline CHANGELOG.md:11]: have to bump `grafanaDependency` in [plugin.json](https://github.com/cognitedata/cognite-grafana-datasource/blob/master/src/plugin.json)
and make it as a breaking change

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 This article documents the ongoing improvements we're making to the **Cognite
 Data Source for Grafana**.
 
+## 4.3.0 - Oct 24th, 2024
```
Comment (polomani) [inline CHANGELOG.md:6]:
```suggestion
## 5.0.0 - Oct 24th, 2024
```

File: package.json [hunk 3]
```diff
@@ -99,10 +99,10 @@
...
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "^10.1.2",
-    "@grafana/runtime": "^10.3.1",
-    "@grafana/schema": "^10.1.2",
-    "@grafana/ui": "^10.1.2",
+    "@grafana/data": "^11.3.0",
+    "@grafana/runtime": "^11.3.0",
+    "@grafana/schema": "^11.3.0",
+    "@grafana/ui": "^11.3.0",
     "deepdash": "^5.0.0",
```
[the remark this answers is not in the document]
Comment (abdullah-cognite) [reply package.json:106]: bumped

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
 This article documents the ongoing improvements we're making to the **Cognite
 Data Source for Grafana**.
 
+## 5.0.0 - Oct 24th, 2024
+
+### Bug fixes
+
+- Upgrade plugin dependencies to fix CVEs
```
Comment (polomani) [inline CHANGELOG.md:10]:
```suggestion
### Bug fixes

- **Breaking** Upgrade plugin dependencies to fix CVEs  (requires Grafana v11)
```

**Review by polomani — COMMENTED** [review]
added a comment for the changelog

**Review by polomani — APPROVED** [review]
added a comment for the changelog

**Pull request merged**